setup.py

```diff
@@ -87,6 +87,7 @@
         "pandas>=1.5,<3",
         "matplotlib>=3.6.0,<4",
         "numpy>=1.24.0,<2",
+        "deprecated"
```

Comment:
Can we pinning it to version latest(`1.2.14`) or a compatible

Reply:
+1 on this